File: Java/program117.java
import java.util.*;

class program117
{
    public static void main(String A[])
    {
       
        int [] Arr = new int [5];
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter elements :");

        Arr[0] = sobj.nextInt();
        Arr[1] = sobj.nextInt();
        Arr[2] = sobj.nextInt();
        Arr[3] = sobj.nextInt();
        Arr[4] = sobj.nextInt();

        System.out.println("Elements from array are :");
        int iCnt = 0;
        for(iCnt = 0; iCnt < 5 ; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }
    }
}

File: C++/program117.cpp
#include<iostream>
using namespace std;
int main()
{
    int Arr[5] = { '\0'};
    int iCnt = 0;

    cout<<"Enter array elements "<<'\n';

    cin>>Arr[0];
    cin>>Arr[1];
    cin>>Arr[2];
    cin>>Arr[3];
    cin>>Arr[4];

    cout<<"Elements from array are "<<'\n';
    for(iCnt = 0; iCnt < 5; iCnt++)
    {
        cout<<Arr[iCnt] <<'\n';
    }
    
    return 0;
}